src/test/java/seedu/address/logic/parser/FindByWorkExperienceCommandParserTest.java

```diff
@@ -16,7 +16,8 @@ public class FindByWorkExperienceCommandParserTest {
     @Test
     public void parse_validArgs_returnsFindCommand() {
         FindByWorkExperienceCommand expectedCommand =
-                new FindByWorkExperienceCommand(new WorkExperienceContainsKeywordsPredicate("Intern", "Google", "2024"));
+                new FindByWorkExperienceCommand(new WorkExperienceContainsKeywordsPredicate("Intern",
```

Comment:
The inclusion of "2024" within the command parser is a great improvement for specificity. However, I recommend adding a few more test cases to ensure that invalid or partial years are caught by this parser, for instance, "202", "24", or non-numerical strings.